statesman-engine/src/main/java/io/appform/statesman/engine/handlebars/HandleBarsHelperRegistry.java

```diff
@@ -566,6 +574,23 @@ private void registerDate() {
         });
     }
 
+    private void registerIncDate() {
+        handlebars.registerHelper("incDate", (Integer context, Options options) -> {
+            try {
+                if (null != context) {
```

Comment:
If context is null, set it to 1, so {{incDate}} will automatically give you next day